src/App.js

```diff
@@ -7,16 +7,8 @@ function App() {
       <header className="App-header">
         <img src={logo} className="App-logo" alt="logo" />
         <p>
-          Edit <code>src/App.js</code> and save to reload.
+          Hello World
```

Comment:
You missed the exclamation mark.

```suggestion
          Hello World!
```